pepper-course-lesson-1.msrt

```diff
@@ -141,7 +141,7 @@
 29
 00:02:14,340 --> 00:02:17,380
 [rus] ...никаких групп больше быть не должно.
-[eng] ...another groups shouldn't be there.
+[eng] ...the parts should not be grouped.
```

Comment:
Why have you translated like that? I made a stress on "Groups" and for me "the parts should not be grouped" is weird after "Such parts should be grouped". But if it's OK, so OK then...

Reply:
I can see what you're saying about it being a bit weird after saying some parts should be grouped together, especially if you miss the line before it. ("But in other cases...") "...another groups shouldn't be there" isn't grammatically correct, but it can be corrected to "...other groups shouldn't be there" or "...another group shouldn't be there", or even better we could drop the other/another because it says already says that it is other cases in the previous line. However to me all of these options imply that there is a group (or groups) that exist, which should not be there (which require ungrouping). By putting it this differently as "...the parts should not be grouped", it instead implies that there are parts that exist, and that you should not group them in the future.